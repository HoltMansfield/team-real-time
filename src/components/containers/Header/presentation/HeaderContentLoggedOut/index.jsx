import React from 'react'


export default function HeaderContentLoggedOut(props) {
  return (
    <div>
      header logged out
    </div>
  )
}
