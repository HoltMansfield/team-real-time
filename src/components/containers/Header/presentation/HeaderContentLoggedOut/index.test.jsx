import React from 'react'
import renderer from 'react-test-renderer'
import HeaderContentLoggedOut from '../HeaderContentLoggedOut'

it('renders correctly', () => {
  const component = renderer
    .create(<HeaderContentLoggedOut />)
    .toJSON()
  expect(component).toMatchSnapshot()
})
