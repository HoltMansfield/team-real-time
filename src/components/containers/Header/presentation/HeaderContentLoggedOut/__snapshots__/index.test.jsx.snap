// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="menu-container"
>
  <div
    className="menu-tmx"
  />
  <div
    className="row menu-tmx-content"
  >
    <div
      className="columns medium-4 large-4"
    >
      <img
        alt="TMX Matrix"
        className="tmxlogo-header"
        src="tmx_matrix_logo_small.png"
        title="TMX Matrix"
      />
    </div>
    <div
      className="columns header-arrow medium-8 large-8"
    >
      <ul
        className="tmx-header-list"
      >
        <li>
          <span
            onClick={undefined}
            onKeyDown={undefined}
            role="button"
            tabIndex="0"
          >
            Sign In
          </span>
          <span
            className="icon-right-arrow"
          />
        </li>
      </ul>
    </div>
  </div>
</div>
`;
